python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -97,8 +104,21 @@ spec:
         - name: autoscaler
           image: rayproject/kuberay-autoscaler
           imagePullPolicy: Always
-          command: ["/bin/sh"]
-          args: ["-c", "$HOME/anaconda3/bin/python /home/ray/run_autoscaler_with_retries.py --redis-password 5241590000000000"]
+          env:
+          - name: RAY_CLUSTER_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: RAY_CLUSTER_NAME
+            # This value must match the metadata.name of the RayCluster CR.
+            # The autoscaler uses this env variable to determine which Ray CR to interact with.
+            # TODO: Match with CR name automatically via operator, Helm, and/or Kustomize.
+            value: raycluster-complete
```

Comment:
Why not
```
valueFrom:
  fieldRef:
    fieldPath: metadata.name
```
here and get rid of the TODO?

Reply:
Ah I see, this is the pod template not the RayCluster spec. Probably the best solution here is to let kuberay define this environment variable. @DmitriGekhtman and @Jeffwan can you make sure this happens? I don't want to end up with a worse solution here just because the code is living in two different repositories.

Reply:
Yep, definitely on the todo to automate formatting of the autoscaler container spec!

Reply:
Resolving thread for my own tracking purposes.